runtime/stdlib/contracts/test.cdc

```diff
@@ -0,0 +1,236 @@
+/// Test contract is the standard library that provides testing functionality in Cadence.
+///
+pub contract Test {
+
+    /// Convenient function to fail a test.
+    /// Is equivalent to calling `assert(false)`.
+    ///
+    pub fun fail() {
+        assert(false)
+    }
+
+    /// Blockchain emulates a real network.
+    ///
+    pub struct Blockchain {
+
+        pub let backend: AnyStruct{BlockchainBackend}
+
+        init(backend: AnyStruct{BlockchainBackend}) {
+            self.backend = backend
+        }
+
+        /// Executes a script and returns the script return value and the status.
+        /// `returnValue` field of the result will be `nil` if the script failed.
+        ///
+        pub fun executeScript(_ script: String, _ arguments: [AnyStruct]): ScriptResult {
```

Comment:
It's more consistent with our existing style to give at least one of the parameters a label when there is more than one. Here, imo, it makes sense to have `arguments` be a labeled parameter.

Reply:
Generally from what I've seen, the label is given to only one parameter, if it is optional (e.g: assert() - the message is optional). Here, ideally, args can be made optional, but we don't have the support for it yet. So, unless the argument is not optional, I feel it's more consistent to either have labels for all or not have at all.

Reply:
Hmm, interesting, I was not aware that `assert` was special cased to have an optional argument. The example I was thinking of was `account.link`, where one of the two paths has a `target` label to make the code more readable.

Reply:
Interesting! I'm all in for adding the label to the 'arguments', if that pattern is used more often than the other way around, and helps the readability 👍 

I guess we need a general rule (for consistency) on when to add labels:
- for just one param
- for none of the params
- for all params / for the one and only param